### DISCUSSION TITLE
如何获取走马灯当前幻灯片的index

### DISCUSSION THREAD

**Discussion by ziyishenlong** [element-plus/element-plus#7161] [Q&A]
我之前使用element-ui时是通过this.$refs.carousel.activeIndex来获取走马灯当前幻灯片的index，所以想问一下在element-plus里如何获取走马灯当前幻灯片的index

**Comment by tolking** [comment]
不是可以通过 chang 事件获取吗 [Demo](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtY2Fyb3VzZWwgaW5kaWNhdG9yLXBvc2l0aW9uPVwib3V0c2lkZVwiIEBjaGFuZ2U9XCJoYW5kbGVDaGFuZ2VcIj5cbiAgICA8ZWwtY2Fyb3VzZWwtaXRlbSB2LWZvcj1cIml0ZW0gaW4gNFwiIDprZXk9XCJpdGVtXCI+XG4gICAgICA8aDM+e3sgaXRlbSB9fTwvaDM+XG4gICAgPC9lbC1jYXJvdXNlbC1pdGVtPlxuICA8L2VsLWNhcm91c2VsPlxuPC90ZW1wbGF0ZT5cblxuPHNjcmlwdCBsYW5nPVwidHNcIiBzZXR1cD5cbiBpbXBvcnQgeyBFbE1lc3NhZ2UgfSBmcm9tICdlbGVtZW50LXBsdXMnXG4gIFxuZnVuY3Rpb24gaGFuZGxlQ2hhbmdlKGluZGV4OiBzdHJpbmcpIHtcbiAgY29uc29sZS5sb2coaW5kZXgpXG4gIEVsTWVzc2FnZSh7XG4gICAgbWVzc2FnZTogYOW9k+WJjWluZGV4OiAke2luZGV4fWAsXG4gICAgdHlwZTogJ3N1Y2Nlc3MnLFxuICB9KVxufVxuPC9zY3JpcHQ+XG5cbjxzdHlsZSBzY29wZWQ+XG4uZWwtY2Fyb3VzZWxfX2l0ZW0gaDMge1xuICBjb2xvcjogIzQ3NTY2OTtcbiAgZm9udC1zaXplOiAxOHB4O1xuICBvcGFjaXR5OiAwLjc1O1xuICBsaW5lLWhlaWdodDogMzAwcHg7XG4gIG1hcmdpbjogMDtcbiAgdGV4dC1hbGlnbjogY2VudGVyO1xufVxuXG4uZWwtY2Fyb3VzZWxfX2l0ZW06bnRoLWNoaWxkKDJuKSB7XG4gIGJhY2tncm91bmQtY29sb3I6ICM5OWE5YmY7XG59XG5cbi5lbC1jYXJvdXNlbF9faXRlbTpudGgtY2hpbGQoMm4gKyAxKSB7XG4gIGJhY2tncm91bmQtY29sb3I6ICNkM2RjZTY7XG59XG48L3N0eWxlPlxuIiwiaW1wb3J0LW1hcC5qc29uIjoie1wiaW1wb3J0c1wiOnt9fSJ9)
